README.md

```diff
@@ -69,6 +69,7 @@ This node does not take any inputs.
 | `phase`       | double | Desired phase of the trigger relative to ToS (Top of Second) |
 | `frame_rate`  | double | Desired frequency of the trigger in Hz                       |
 | `cpu_core_id` | int    | Desired CPU core for execution\*                             |
+| `pulse_width_ms` | int | Desired pulse width (i.e., time period that GPIO pin to be high) in millisecond |
```

Comment:
Suggest: "Desired pulse width (the time period in milliseconds for the GPIO pin to remain high)"

Reply:
@drwnz 
Thank you for your suggestion! I've applied your suggestion at c15de195b648737c8efc9e66483a18b3da0b8122